Material.Blazor/Components/FloatingActionButton/MBFloatingActionButton.razor.cs

```diff
@@ -92,7 +92,8 @@ protected override async Task OnInitializedAsync()
 
             ConditionalCssClasses
                 .AddIf("mdc-fab--mini mdc-fab--touch", () => Type == MBFloatingActionButtonType.Mini)
-                .AddIf("mdc-fab--extended", () => Type == MBFloatingActionButtonType.Extended);
+                .AddIf("mdc-fab--extended", () => Type == MBFloatingActionButtonType.Extended)
```

Comment:
Can `Extended` and `Mini` both be used, according to spec?

Reply:
They're mutually exclusive.

https://github.com/material-components/material-components-web/tree/master/packages/mdc-fab#floating-action-buttons